### PR TITLE
fix(#46) - Removing images from README.md for opc-footer and from demo folder and other fixes

### LINKED ISSUE #46: Changes in generator README, adding npmignore and CleanWebpackPlugin
## Is your chore related to a problem? Please describe 

Removing images from the demo folder and README

## Describe alternatives you've considered

None

### PR DESCRIPTION
**Pull request by hybridx**
### Resolves #46 

# Explain the feature/fix

Removed images and also eslint pre hooks

## Does this PR introduce a breaking change

Yes

- Changed opc-footer attribute opc-links-groups to opc-links

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my own code
- [x] I have documented my code, particularly in areas like todo, complex logic, quick fix, temporary patch, etc.
#### If it is a new component
- [ ] Does your component follow One Platform style guidelines?
- [ ] Does your component web accessibility standards? [Helper Doc](https://www.w3.org/TR/wai-aria-1.1/)
- [ ] Does your component support desktop screen sizes (350px, 720px, 1150px ,1920px)
#### Browsers you have tested in
- [ ] Latest two versions of Mozilla Firefox and Google Chrome supported by Red Hat Enterprise Linux distribution
- [ ] Google Chrome [Latest 2 versions]
- [ ] Mozilla Firefox [Latest 2 versions]
- [ ] Microsoft Edge [Latest 2 versions]
- [ ] Apple Safari [Latest 2 versions]
- [ ] Microsoft Internet Explorer 11

### CODE REVIEW

File: packages/opc-footer/README.md [hunk 1]
```diff
@@ -14,24 +14,20 @@ npm install
 
 ## Usage
 
-the opc-footer have supports dark theme, by default the light theme is selected. for example
-footer with default theme:
+The opc-footer defaults to white background a.k.a light theme and supports dark theme as well.
```
Comment (diwanshi) [inline packages/opc-footer/README.md:17]: @hybridx can u follow same readme structure as opc-header for this component.

File: packages/opc-footer/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@one-platform/opc-footer",
-  "version": "0.0.1-prerelease",
+  "version": "0.0.2-prerelease",
   "description": "A simple web component built in javascript and lit",
```
Comment (diwanshi) [inline packages/opc-footer/package.json:4]: @hybridx @gisumit please provide proper description in package.json

File: packages/opc-footer/src/opc-footer.ts [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
 import style  from './opc-footer.scss';
-import { OpcFooterLinksGroup } from './opc-footer-interface';
 
 @customElement('opc-footer')
 export class OpcFooter extends LitElement {
   @property({ attribute: 'opc-theme' }) theme: string = 'light';
 
-  @property({ type: Array, attribute: 'opc-links-group' })
+  @property({ type: Array, attribute: 'opc-links' })
   linksGroups: OpcFooterLinksGroup[] | undefined = [];
```
Comment (diwanshi) [inline packages/opc-footer/src/opc-footer.ts:9]: if you're changing the name in the property then make the change everywhere @hybridx @gisumit

**Review by diwanshi — CHANGES_REQUESTED** [review]

File: packages/opc-footer/README.md (hunk 1 above)
Comment (hybridx) [reply packages/opc-footer/README.md:17]: Yes I'll add the change in the generator as well

File: packages/opc-footer/src/opc-footer.ts (hunk 3 above)
Comment (hybridx) [reply packages/opc-footer/src/opc-footer.ts:9]: @diwanshi should we make it to just `links` ?

**Comment by hybridx** [comment]
Closing this PR as it duplicates #49

**Pull request closed without merging**